fronts-client/src/lib/dnd/Level.tsx

```diff
@@ -206,12 +206,14 @@ class Level<T> extends React.Component<Props<T>, State> {
 
 		if (!isInside(this.props.parents, [type, id])) {
 			const [parentType, parentId] = parents[parents.length - 1];
-			const from = !forceClone &&
-				isMove(this.props.parents, parents) && {
-					type: parentType,
-					id: parentId,
-					index,
-				};
+			const from =
+				forceClone && isMove(this.props.parents, parents)
+					? {
+							type: parentType,
+							id: parentId,
+							index,
+						}
+					: null;
```

Comment:
This refactor allows us to always have the from type as 

```	from: null | PosSpec;```
as opposed to 
```from: false | PosSpec;``` and then coercing false to null elsewhere